docs/user-guide/src/version_support.md

```diff
@@ -34,14 +37,19 @@ Since BMO and Ironic do not follow similar release cycles they are backward comp
 - capm3-v0.5.1
 - capm3-v0.5.0
 
+Up until `capm3-v1.1.2` tag, BMO follows the same trend as Ironic. However, starting after `capm3-v1.1.2` and forwards tag, BMO follows the semantic versioning scheme for its own release cycle, the same way as CAPM3 and IPAM. At the moment, we always cut rolling releases from the main branch of BMO and tag them with the release version (i.e: v0.1.X). Here are available and supported BMO image tags:
```

Comment:
```suggestion
Up until `capm3-v1.1.2` tag, BMO follows the same trend as Ironic. However, since `capm3-v1.1.2`, BMO follows the semantic versioning scheme for its own release cycle, the same way as CAPM3 and IPAM. At the moment, we always cut rolling releases from the main branch of BMO and tag them with the release version (i.e: v0.1.X). Here are available and supported BMO image tags:
```